smartsim/_core/mli/infrastructure/control/workermanager.py

```diff
@@ -24,14 +24,25 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-import multiprocessing as mp
+import sys
+
+# isort: off
+try:
+    import dragon
+    from dragon import fli
+except ImportError as exc:
+    if not "pytest" in sys.modules:
```

Comment:
are you sure you want to look for pytest here? might be a copy/paste mistake

Reply:
It was an attempt to avoid failing tests when dragon was not available. This will be fixed once #621 will go in